ui/components/app/confirm/sender-info/sender-info.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { memo, useEffect, useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { MESSAGE_TYPE } from '../../../../../shared/constants/app';
+import {
+  BackgroundColor,
+  BorderRadius,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../../selectors';
+import { Box } from '../../../component-library';
+import {
+  ConfirmInfo,
+  ConfirmInfoRowConfig,
+  ConfirmInfoRowType,
+} from '../info/info';
+
+const SenderInfo: React.FC = memo(() => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(currentConfirmationSelector);
+  const [rowConfigs, setRowConfigs] = useState<ConfirmInfoRowConfig[]>([]);
+
+  useEffect(() => {
+    if (
```

Comment:
is there a way to unit test that rowConfigs is set/not set?

Reply:
Yep let me add that test

Reply:
done